app/controllers/onboarding_controller.rb

```diff
@@ -46,10 +44,8 @@ def verify_token
     raise ActionController::BadRequest if invalidated_jti.exists?
 
     JsonWebToken.decode(jwt_param)
-  rescue ActionController::BadRequest
-    redirect_to onboarding_unauthorized_path
-  rescue JWT::DecodeError
-    redirect_to onboarding_unauthorized_path
+  rescue ActionController::BadRequest, JWT::DecodeError
+    render :unauthorized, status: :unauthorized
```

Comment:
Nice! I like this! But I got confused cause the `onboarding#create_invite_url` is also a `post` request, so wouldn't your comment :point_up: be relevant?

Reply:
Maybe it doesn't need to be a `post` request?

Reply:
You’re referring to my comment about always redirecting from a post to a get route to prevent resubmitting the post request? If so, I don’t think it applies to the `create_invite_url` action, as that’s not a user-facing route, but an API endpoint.

Reply:
yeah, true... wasn't thinking about that.